sysutils/smartmontools/Portfile

```diff
@@ -31,6 +31,13 @@ configure.args      --without-savestates \
                     --enable-sample \
                     --with-libcap-ng=no
 
+platform darwin {
+    if {${configure.cxx_stdlib} eq "libstdc++"} {
```

Comment:
Shouldn't this use `[string match "*libstdc\+\+" ${configure.cxx_stdlib}]`? When I checked earlier today if using libstdc++ was a viable workaround noticed that "base" now selects "macports-libstdc++" when you pass `configure.compiler=macports-gcc-N`.

If I understand the raison d'être for that approach (avoid confusion between the system libstdc++ and the MacPorts copy) it should be used on legacy, libstdc++-based systems too.

Reply:
I copied and pasted that directly out of your recommendation in the other PR. You're killing me here, Rene.